common/build.gradle

```diff
@@ -158,6 +158,7 @@ dependencies {
 
     implementation 'org.apache.httpcomponents.core5:httpcore5:5.3'
     implementation "com.nimbusds:nimbus-jose-jwt:$rootProject.ext.nimbusVersion"
+    implementation "androidx.activity:activity:1.8.2"
```

Comment:
picked 1.8.2 as suggested by @nickbopp 
[Pull request 4266808: Downgrade required androidx.activity version to 1.8 - Repos](https://office.visualstudio.com/OneAuth/_git/OneAuth/pullrequest/4266808)